packages/channels/botpress-channel-web/src/views/web/index.jsx

```diff
@@ -40,7 +40,17 @@ const defaultOptions = {
   showUserName: false,
   showUserAvatar: false,
   botConvoTitle: 'Botpress Webchat',
-  enableTranscriptDownload: false
+  enableTranscriptDownload: false,
+  customStylesheet: ''
+}
+
+const injectDOMElement = (tagName, targetSelector, options = {}) => {
+  const element = document.createElement(tagName)
+
+  Object.keys(options).forEach(key => (element[key] = options[key]))
```

Comment:
You should also be able to just `Object.assign(element, options)` (but please check)